website/docs/r/sqs_queue_policy.html.markdown

```diff
@@ -34,7 +34,7 @@ resource "aws_sqs_queue_policy" "test" {
       "Resource": "${aws_sqs_queue.q.arn}",
       "Condition": {
         "ArnEquals": {
-          "aws:SourceArn": "${aws_sqs_queue.q.arn}"
+          "aws:SourceArn": "${aws_sns_topic.topic_name_here.arn}"
```

Comment:
Hi @dasiths - thanks for catching this and submitting this fix! 

Just one minor nitpick here - our convention is to use `example` for names in documentation, like this: 
```suggestion
          "aws:SourceArn": "${aws_sns_topic.example.arn}"
```

But once that's changed we can get this merged! Thank you! 👍